src/objects/weapons/MachineGun.js

```diff
@@ -1,16 +1,16 @@
-import AbstractWeapon from "./AbstractWeapon.js";
-import Ball from "../Ball.js";
+import AbstractWeapon from './AbstractWeapon';
+import Bullet from '../Bullet';
 
 export default class MachineGun extends AbstractWeapon {
-    constructor(player) {
-        super(player)
-    }
+  constructor(player) {
+    super(player)
+  }
 
-    fire() {
-        const bullet = new Ball(this.player.scene, this.player.x, this.player.y);
-    }
-    
-    fireRelease() {
-        this.firstShot = false;
-    }
+  fire() {
+    const bullet = new Bullet(this.player.scene, this.player.x, this.player.y);
+  }
+
+  fireRelease() {
```

Comment:
Can remove this as it inherits this function from abstract weapon class. 
Only need to add this function is you want to overwrite the original function.